### PR TITLE
Package ocamlformat.0.9.1

### PR DESCRIPTION
**Pull request by gpetiot**
### `ocamlformat.0.9.1`
Auto-formatter for OCaml code
OCamlFormat is a tool to automatically format OCaml code in a uniform style.

---
* Homepage: https://github.com/ocaml-ppx/ocamlformat
* Source repo: git+https://github.com/ocaml-ppx/ocamlformat.git
* Bug tracker: https://github.com/ocaml-ppx/ocamlformat/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by gpetiot** [comment]
See https://github.com/ocaml-ppx/ocamlformat/issues/891

File: packages/ocamlformat/ocamlformat.0.9.1/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "OCamlFormat Team <ocamlformat-team@fb.com>"
+authors: "Josh Berdine <jjb@fb.com>"
+homepage: "https://github.com/ocaml-ppx/ocamlformat"
+bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
+dev-repo: "git+https://github.com/ocaml-ppx/ocamlformat.git"
+url {
+  src: "https://github.com/ocaml-ppx/ocamlformat/archive/0.9.tar.gz"
```
Comment (Julow) [inline packages/ocamlformat/ocamlformat.0.9.1/opam]: Should be https://github.com/ocaml-ppx/ocamlformat/archive/0.9.1.tar.gz

Comment (gpetiot) [reply packages/ocamlformat/ocamlformat.0.9.1/opam]: Fixed.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**